react/features/local-recording/controller/RecordingController.js

```diff
@@ -21,6 +21,20 @@ const COMMAND_START = 'localRecStart';
  */
 const COMMAND_STOP = 'localRecStop';
 
+/**
+ * One-time command used to trigger the moderator to resend the commands.
+ * This is a workaround for newly-joined clients to receive remote presence.
+ */
+const COMMAND_PING = 'localRecPing';
```

Comment:
This makes me want to cry :(

Reply:
To clarify: I don't want you to change it, I'm just sad that we have to resort to solutions like this.

Reply:
I agree there should be a better way. I don't fully understand https://github.com/jitsi/lib-jitsi-meet/blob/98acf1336df7a6fe6fc27c16c829ef607ec20ceb/modules/xmpp/ChatRoom.js#L394 yet.